lib/modules/login/repo/login_repo.dart

```diff
@@ -1 +1,50 @@
+import 'package:dartz/dartz.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:google_sign_in/google_sign_in.dart';
 
+import '../../../_utils/entities/api_response.dart';
+import '../model/login_dto.dart';
+
+abstract class LoginRepo {
+  Future<APIResponse<LoginDTO>> googleLogin();
+  Future<APIResponse<Unit>> googleLogout();
+}
+
+class LoginRepoImp implements LoginRepo {
+  final FirebaseAuth _firebaseAuth = FirebaseAuth.instance;
+
+  @override
+  Future<APIResponse<LoginDTO>> googleLogin() async {
```

Comment:
Put some spacing between statements where it required... It seems like a paragraph is written

Reply:
Resolved